src/shims/unix/foreign_items.rs

```diff
@@ -315,6 +315,41 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
                     }
                 }
             }
+            "aligned_alloc" => {
+                let [align, size] =
```

Comment:
```suggestion
                // This is a C11 function, we assume all Unices have it.
                let [align, size] =
```

Reply:
well windows kind of support it but it s different. `_aligned_alloc` but you need to call `_aligned_free`, kind of complicated ..

Reply:
Yeah, so they don't support the official aligned_alloc which says you can free with `free`. "kind of" support isn't support. ;)